pygmt/figure.py

```diff
@@ -145,7 +145,7 @@ def psconvert(self, icc_gray=False, **kwargs):
         PDF, PNG, PPM, SVG, TIFF) using GhostScript.
 
         If no input files are given, will convert the current active figure
-        (see :func:`pygmt.figure`). In this case, an output name must be given
+        (see :func:`pygmt.Figure`). In this case, an output name must be given
```

Comment:
Strictly speaking, `pygmt.Figure` is a class, not a function.
```suggestion
        (see :class:`pygmt.Figure`). In this case, an output name must be given
```

Reply:
Thanks @seisman! I made this change alread in PR #2043 https://github.com/GenericMappingTools/pygmt/pull/2043/commits/ef169ec9e33c5b37d35e790783526f336734ed54.